cmd/conduit/root/root.go

```diff
@@ -76,7 +76,7 @@ func (c *RootCommand) SubCommands() []ecdysis.Command {
 		&config.ConfigCommand{RunCmd: runCmd},
 		&initialize.InitCommand{Cfg: &runCmd.Cfg},
 		&version.VersionCommand{},
-		&pipelines.PipelinesCommand{},
-		&run.RunCommand{},
+		&pipelines.PipelinesCommand{RunCmd: runCmd},
```

Comment:
I guess you're trying to inherit the flags from the run command? That won't work, since the pipelines command is not a subcommand of run. It would only work if `RootCommand` had the flags attached and they were marked as persistent (see the [example](https://github.com/ConduitIO/ecdysis/blob/4a9a7e14d5e156daec9e193cd6116343e8eb7d0e/example/main.go#L71-L72)).

Reply:
@lovromazgon I was actually trying to read from the configured port. Haven't been able to test this properly because of this bug, so I might try to fix it to ensure this is working as expected https://github.com/ConduitIO/conduit/issues/2062.

Reply:
It is a bit awkward currently to share the same set of flags in unrelated commands (commands that don't share the same parent), or when commands do share the same parent, but the parent doesn't use the same flags. Maybe this can be addressed in ecdysis so it's easier to achieve, but I'm not sure how 🤔